sqlsynthgen/json_schemas/config_schema.json

```
@@ -76,7 +76,16 @@
             "type": "string"
           },
           "args": {
-            "type": "object"
+            "type": [
+              "array",
+              "null"
```

```suggestion
              "array"
```

And so on for the others

---

Makes sense! I already pushed the changes